Watchman.Discord/Areas/Administration/Controllers/AdministrationController.cs

```diff
@@ -86,34 +86,24 @@ public async Task ReadUserMessages(DiscordRequest request, Contexts contexts)
         [DiscordCommand("set role")]
         public async Task SetRoleAsSafe(DiscordRequest request, Contexts contexts)
```

Comment:
do przepisania na IBotCommand

Reply:
a to chyba dopiero kiedy do IBotCommand dojdzie IEnumerable, bo inaczej to chyba się nie da?

Reply:
zawsze można dodać ienumerable przy okazji tego zadania

to będzie działało tak
że jak jest ienumerable
to sprawdzasz kolejne argumenty do czasu, kiedy name != null 

przy czym każdy sprawdzasz pod kątem typu danych, odpalasz konwersje itd